build.gradle

```diff
@@ -16,4 +16,20 @@ allprojects {
 
     group = GROUP_ID
     version = VERSION_NAME
+
+    tasks.withType(JavaCompile) {
+        options.fork = true
+	options.forkOptions.jvmArgs += [
```

Comment:
Add a comment on why this is needed.

Reply:
Done, clarified that this is required for errorprone's reflection.